packages/next-app/src/app/daoadmin/(components)/DaoAdminSidebar.tsx

```diff
@@ -0,0 +1,103 @@
+import * as SelectPrimitive from "@radix-ui/react-select";
+import Link from "next/link";
+import { useState } from "react";
+
+import { Badge } from "#/components/Badge";
+import { Select, SelectItem } from "#/components/Select";
+import Sidebar from "#/components/Sidebar";
+import { ActionAttribute, useAdminTools } from "#/contexts/AdminToolsContext";
+import { hardcodedData } from "#/utils/hardcodedData";
+import { truncateAddress } from "#/utils/truncateAddress";
+
+interface IFilter {
+  name: string;
+  options: string[];
+}
+
+export function DaoAdminSidebar() {
+  const data = hardcodedData;
+  const [querySearch, setQuerySearch] = useState("");
+  const [selectedFilter, setSelectedFilter] = useState<string>("");
+  const { selectedAction, handleSetAction } = useAdminTools();
+
+  const handleSearchChange = (e: React.FormEvent<HTMLInputElement>) => {
+    setQuerySearch(e.currentTarget.value);
+  };
+
+  const filteredActionsByTag =
+    selectedFilter !== ""
+      ? data.actions.filter(
+          (item) => item.operationResponsible === selectedFilter
+        )
+      : data.actions;
+
+  const filteredActionsByName = filteredActionsByTag.filter((item) =>
+    item.name.toLowerCase().includes(querySearch.toLowerCase())
+  );
+
+  const filters: IFilter[] = [
+    {
+      name: "Operation Responsible",
+      options: ["Maxis", "Labs"],
+    },
+  ];
+
+  return (
+    <Sidebar>
+      <Sidebar.Header name="DAO Actions">
+        <div className="flex justify-between">
+          <Sidebar.InputFilter
+            placeHolder="Search for DAO Action"
+            onChange={handleSearchChange}
+          />
+          <Select
+            onValueChange={(value: string) => setSelectedFilter(value)}
+            theme={"light"}
+          >
+            {filters.map((filter) => (
+              <SelectPrimitive.Group key={filter.name}>
+                <SelectPrimitive.Label className="py-1 px-2 text-sm text-gray-600">
+                  {filter.name}
+                </SelectPrimitive.Label>
+                <SelectItem value="" theme={"light"}>
+                  No filter
+                </SelectItem>
+                {filter.options.map((option) => (
+                  <SelectItem key={option} value={option} theme={"light"}>
+                    {option}
+                  </SelectItem>
+                ))}
+              </SelectPrimitive.Group>
```

Comment:
Wouldn't it make sense to expose this through the Select component instead of having both here?